### DISCUSSION TITLE
Save project in EnMAP

### DISCUSSION THREAD

**Discussion by dpoursanidis** [EnMAP-Box/enmap-box#305] [Ideas]
Working with EnMAP, what i feel missing is to save the project as i save a QGIS project.
This could be to save it as geojson file as ENVI does and you load it and all data come back.
Or as QGIS project (?) and restore all data in EnMAP?
I have seen similar idea/issue in the near past but couldn't locate it.

**Comment by janzandr** [comment]
We are already working on it. Basic functionality is implemented:
https://github.com/EnMAP-Box/enmap-box/issues/201
